app/static/css/dashboard.css

```diff
@@ -1,225 +1,182 @@
+    
+    -o-transform     : none;
+    -moz-transform   : none;
+    -webkit-transform: none;
+    transform        : none;
+    
+    -webkit-animation: none;
+    -moz-animation   : none;
+    -o-animation     : none;
+    animation        : none;
+}
 
 @media print {
+    .wrapper {
+        display: block;
+    }
+    
+    #sidebar {
+        display   : none;
+        width     : 0px;
+        max-height: 0px;
+    }
+    
+    h2#name {
+        color: green;
+    }
+    
+    input.form-control::-webkit-input-placeholder {
+        opacity: 0;
+    }
+    
+    input.form-control::-moz-placeholder {
+        opacity: 0;
+    }
+    
+    input.form-control::-ms-placeholder {
+        opacity: 0;
+    }
+    
+    input.form-control::placeholder {
+        color: transparent;
+    }
+}
 
-  #sidebar {
-    display: none;
-    width: 0px;
-    max-height: 0px;
-  }
-  .wrapper {
-    display: block;
-  }
-  h2#name {
-    color: green;
-  }
- 
-
-  input.form-control::-webkit-input-placeholder {
-    opacity: 0;
-  } 
-  input.form-control::-moz-placeholder {
-      opacity: 0;
-  }
-  input.form-control::-ms-placeholder {
-      opacity: 0;
-  }
-  input.form-control::placeholder {
-           color: transparent;
-  }
-
+@media (max-width: 768px) {
+    .wrapper {
+        display: block;
+    }
+    
+    #sidebar {
+        min-width : auto;
+        max-width : none;
+        min-height: 0px;
+    }
+    
+    #sidebar.inactive {
+        margin-left: -1000px;
+        height     :0px;
+    }
+
+    .navbar span.pagename {
+        display: none;
+    }
 }
 
 .wrapper {
-    display: flex;
-    flex-wrap: wrap; /* For narrow screens*/
+    display    : flex;
+    flex-wrap  : wrap;
     align-items: stretch;
 }
 
 #sidebar {
-    flex: 1 0 160px;
-    min-width: 160px;
-    max-width: 160px;
+    flex      : 1 0 160px;
+    min-width : 160px;
+    max-width : 160px;
     min-height: 100vh;
-    font-size: .9em;
+    font-size : .9em;
     box-shadow: inset -1px 0 0 rgba(0, 0, 0, .1);
     transition: all 0.3s;
 }
 
-.disable-animations, .disable-animations * {
-  /* CSS transitions */
-  -o-transition-property: none !important;
-  -moz-transition-property: none !important;
-  -webkit-transition-property: none !important;
-  transition-property: none !important;
-  /* CSS transforms */
-  -o-transform: none !important;
-  -moz-transform: none !important;
-  -webkit-transform: none !important;
-  transform: none !important;
-  /* CSS animations */
-  -webkit-animation: none !important;
-  -moz-animation: none !important;
-  -o-animation: none !important;
-  animation: none !important;
-}
-
 #sidebar.inactive {
-    /*margin-left: -160px;*/
-    /*display: none;*/
     margin-left: -160px;
-
 }
 
 #sidebar .nav-link {
-  font-weight: 500;
-  color: #333;
+    font-weight: 500;
+    color      : #333;
 }
 
 #sidebar i.fa {
-  font-size: .75rem;
-  margin-right: 3px;
-  color: #999;
+    font-size   : .75rem;
```

Comment:
This whole file:  Two spaces is de facto; colons should have no spaces to their left.

Proper:
```css
#sidebar .nav-link {
  font-size: 0.75rem;
}
```

Reply:
Sorry about that ... I think that this (and the missing newlines) may have happened because I was editing the files using two different text editors, which have different space/tab and end-of-file settings.  Seen it happen before for the same reason.  Will fix.